src/wq/core/task.py

```diff
@@ -270,10 +276,10 @@ def _do_enqueue(self):
                             result_ttl=self.success_ttl(),
                             failure_ttl=self.failure_ttl())
         except Exception as e:
-            print(e)    # TODO log error
+            log().error(e)
             if recover_from_enqueueing_failure():
-                # TODO log msg to say you'll run this task on the spot
-                # last ditch attempt, but only if configured to do so.
+                msg="you'll run this task on the spot last ditch attempt, but only if configured to do so"
```

Comment:
@Necravisaketi can we change the message just slightly? Like: 

> This task could not be added to the work queue, QuantumLeap will try running this task synchronously if WQ_RECOVER_FROM_ENQUEUEING_FAILURE = true.

thanks so much!